.vscode/settings.json

```diff
@@ -6,8 +6,17 @@
   "files.trimTrailingWhitespace": true,
   "files.insertFinalNewline": true,
   "files.trimFinalNewlines": true,
-  "ruby.useLanguageServer": true,
-  "ruby.rubocop.executePath": "Library/Homebrew/shims/gems/",
+  "rubyLsp.rubyVersionManager": {
+    "identifier": "custom"
+  },
+  "rubyLsp.customRubyCommand": "PATH=${workspaceFolder}/Library/Homebrew/vendor/portable-ruby/current/bin:$PATH",
```

Comment:
I wonder if we could even have a shim for the custom Ruby command that automatically installs the gems for you?